### PR TITLE
Feat/tvl tokens

### PR DESCRIPTION
**Pull request by hochbruj**
https://trello.com/c/klHS1Fl7

Additions to that task:
- Added 24h volume to token table
- Filtered out tokens without usd amount from token list
- Changed currency format to 2 decimals for tvl and volume amounts (price with 4 decimals unchanged)

### CODE REVIEW

File: pages/index.js
Comment (OliverNChalk) [inline pages/index.js]:
```suggestion
          ...acc,
          [token0]: {
            ...curr.token0,
            tvl: (acc[token0]?.tvl ?? 0) + reserve0Usd,
            volume: (acc[token0]?.volume ?? 0) + volume0,
          },
          [token1]: {
            ...curr.token1,
            tvl: (acc[token1]?.tvl ?? 0) + reserve1Usd,
            volume: (acc[token1]?.volume ?? 0) + volume1,
```

File: pages/index.js
Comment (OliverNChalk) [inline pages/index.js:99]: New line before return is nice for readability

**Comment by xenide** [comment]
image

In landscape mode on mobile, the justification seems to depend on the length of the name. Should we standardize it to left justify?

**Review by KennethAshley — APPROVED** [review]
Oliver has commented on the issues I'd bring up. Once fixed you have my approval.

**Comment by hochbruj** [comment]
Implemented Oli's review comments.

File: pages/index.js
Comment (OliverNChalk) [inline pages/index.js]: Do we want to leave old code as a comment, does it need to be brought back in the future or why is it here?

**Review by OliverNChalk — APPROVED** [review]

File: pages/index.js
Comment (hochbruj) [reply pages/index.js]: I removed it

**Pull request merged**
